scripts/release-helm.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# This file is open source software, licensed to you under the terms
+# of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+# distributed with this work for additional information regarding copyright
+# ownership.  You may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -euo pipefail
+
+CHANNEL="${CHANNEL:-"$(scripts/release-channel.sh)"}"
+VERSION="${VERSION:-"$(scripts/version.sh)"}"
+
+if [[ "$CHANNEL" == "nightly" ]]; then
+  LATEST_TAG=$(git describe --tags --always --abbrev=0)
```

Comment:
same here, I have a PR that consistently sets these in a makefile and we could reuse that logic if you want https://github.com/scylladb/scylla-operator/pull/365/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R19